src/lib/event.ts

```diff
@@ -40,15 +40,17 @@ export type EventHandler<C = unknown, S = unknown> =
 
 export async function handleEvent(definition: EventHandler, request: RawRequest, context: LambdaContext) {
     if ('headers' in request) {
-        return handleHttpEvent(definition, request, context)
+        if ('http' in definition) {
```

Comment:
if we check for `definition.http !== undefined`, we can skip the `notImplemented in the `handleHttpEvent`, right?